erts/emulator/beam/erl_map.h

```diff
@@ -74,7 +74,7 @@ typedef struct flatmap_s {
 #define flatmap_get_size(x)          (((flatmap_t*)(x))->size)
 
 #ifdef DEBUG
-#define MAP_SMALL_MAP_LIMIT    (3)
+#define MAP_SMALL_MAP_LIMIT    (4)
```

Comment:
:) isn't this cheating? Maybe rewrite the info function to use erts_map_from_ks_and_vs?